src/mcp/tools/core/index.ts

```diff
@@ -14,10 +14,12 @@ import { list_projects } from "./list_projects";
 import { consult_assistant } from "./consult_assistant";
 import { login } from "./login";
 import { logout } from "./logout";
+import { validate_rules } from "./validate_rules";
 
 export const coreTools: ServerTool[] = [
   login,
   logout,
+  validate_rules,
```

Comment:
interesting corner case for feature detection: we should only activate this tool if one of the three services that use it are active.